owner/src/main/java/org/the_chance/honeymart/ui/addCategory/ListCategoryImages.kt

```diff
@@ -0,0 +1,32 @@
+package org.the_chance.honeymart.ui.addCategory
+
+import org.the_chance.design_system.R
+
+data class ListCategoryImages(
```

Comment:
why do we have a data class that contains "List" in its name?
you can rename it to be CategoryImage and just create a list of it.